specification/purview/resource-manager/Microsoft.Purview/preview/2024-04-01-preview/purview.json

```diff
@@ -1631,6 +1630,64 @@
         }
       }
     },
+    "AccountMergeInfo": {
+      "description": "The public Account Merge Info model.",
+      "type": "object",
+      "properties": {
+        "accountLocation": {
```

Comment:
[ARMBlocking] how are you going to do RBAC permissions check on this?

Reply:
This is purely for reporting and informational purposes. Calls will not be performed on the account listed inside of the Account Merge Info.
To overshare, this object contains information from us merging two Purview accounts data. The account location, name, resource group, and subscription id properties are for us to know which account this account merged with. It is solely for informational purposes.

Reply:
But how, for instance, do you avoid spoofing?

Reply:
(BTW thanks for clarifying that these properties are mostly readonly, maybe that answers the concern?)

Reply:
Yes, through this API, they will only be read only. This information is only exposed to users within the same tenant where they would also have knowledge of this account information. I don't believe there should be any concerns with these fields as they are only for information. So when an account is loaded after a merge, the UX can display which account it was merged to.